tests/conftest.py

```
@@ -1,68 +1,88 @@
+        for item in items:
+            if "slow" in item.keywords:
+                item.add_marker(skip_slow)
+
+
+@pytest.fixture
+def test_settings() -> StareSettings:
+    """StareSettings pointing at a test base URL."""
+    return StareSettings(
+        base_url="https://test-glance.example.com/api",
+        auth_url="https://auth.example.com/auth",
+        token_url="https://auth.example.com/token",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "token_url"

(S106)